shrinkwrap.py

```diff
@@ -21,6 +21,86 @@ def get_args():
     return parser.parse_args()
 
 
+def get_machine_count(applications_to_deploy, subordinates):
+    machineCount = 0
+    for appname, app in applications_to_deploy.items():
```

Comment:
The aforementioned problem seems to happen when you pass in a bundle (from line 38).  The `applications_to_deploy` will look like this:
```
Description: foo
Relations: [[bar, baz], [wat, hmm]]
applications: {'easyrsa': {..., NumUnits: 1, ...}
```

So you'll need to parse the bundle better here, or alternatively, only pass `bundle['applications']` to this method on line 38.